frontend/src/scenes/insights/Insight.tsx

```diff
@@ -301,12 +302,14 @@ export function Insight({ insightId }: { insightId: InsightShortId | 'new' }): J
                 }
             />
 
-            {actorOnEventsQueryingEnabled ? (
+            {actorOnEventsQueryingEnabled && alertVisible ? (
                 <div className="mb-4">
-                    <AlertMessage type="info">
-                        Results looking different? We've changed how insights are calculated. Read more about this{' '}
-                        <Link to={`https://posthog.com/docs/how-posthog-works/queries`}>here</Link>. Reach out to us if
-                        you see something unexpected{' '}
+                    <AlertMessage type="info" onClose={() => setAlertVisible(false)}>
```

Comment:
This is going to re-appear on page refresh. We probably want it to go into local storage, so once a users dismisses it, it doesn't come back. Give me one min, I can add a `isCloseSticky` prop to AlertMessage.

Reply:
Oh i thought we want it to come back on refresh

Reply:
Oh you meant "sticky" as in it doesn't return 🤦🏻

Reply:
Hmm, that feels annoying to me. I'd imagine this is the type of thing that most users don't care about, and the ones that do care only need to read once. Is that how you're thinking about it?

Reply:
Yeah I agree, I wasn't thinking too much of it when I first read your suggestion and I confused myself

Reply:
Are you adding the prop

Reply:
Had a chat in Slack on this and we landed on just making it not-dismissible now.